docs/specification/draft/client/elicitation.mdx

```diff
@@ -49,18 +56,115 @@ Clients that support elicitation **MUST** declare the `elicitation` capability d
 ```json
 {
   "capabilities": {
-    "elicitation": {}
+    "elicitation": {
+      "form": {},
+      "oob": {}
+    }
   }
 }
 ```
 
+Clients declaring the `elicitation` capability **MUST** support at least one mode (`form` or `oob`).
+
+Servers **MUST NOT** send elicitation requests with modes that are not supported by the client.
+
 ## Protocol Messages
 
-### Creating Elicitation Requests
+### Elicitation Requests
+
+To request information from a user, servers send an `elicitation/create` request.
+
+All elicitation requests **MUST** include the following parameters:
+
+| Name      | Type   | Options       | Description                                                        |
+| --------- | ------ | ------------- | ------------------------------------------------------------------ |
+| `mode`    | string | `form`, `oob` | The mode of the elicitation.                                       |
+| `message` | string |               | A human-readable message explaining why the interaction is needed. |
+
+The `mode` parameter specifies the type of elicitation:
+
+- `"form"`: In-band structured data collection with optional schema validation. Data is exposed to the client.
+- `"oob"`: Out-of-band interaction via URL navigation. Data is **not** exposed to the client.
+
+### Form Elicitation Requests
```

Comment:
Note to reviewers: This section is not new, but re-organized so that we can clearly describe form elicitation here, and out-of-band elicitation in the next sub-section.